build/helm/keylime/charts/keylime-agent/templates/daemonset.yaml

```diff
@@ -167,6 +172,7 @@ spec:
   selector:
     matchLabels:
       {{- include "agentplugin.selectorLabels" . | nindent 6 }}
+  serviceAccountName: {{ include "agent.serviceAccountName" . }}
```

Comment:
`serviceAccountName` should be part of the spec, and I can see it already below here. Or is there something new going on which I'm missing?